roles/servicetelemetry/templates/manifest_servicemonitors.j2

```diff
@@ -0,0 +1,57 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  labels:
+    app: smart-gateway
+  name: '{{ meta.name }}-sg'
+  namespace: '{{ meta.namespace }}'
+spec:
+  endpoints:
+    - interval: {{ servicetelemetry_vars.backends.metrics.prometheus.scrape_interval }}
+      metricRelabelings:
+        - action: labeldrop
+          regex: pod
+          sourcelabels: []
+        - action: labeldrop
+          regex: namespace
+          sourcelabels: []
+        - action: labeldrop
+          regex: instance
+          sourcelabels: []
+        - action: labeldrop
+          regex: job
+          sourcelabels: []
```

Comment:
These are from a day before I worked on the project. I wonder if we should just try dropping these and building a new set of `metricRelabelings` as we determine? We might end up with the same thing, but at least we'll have worked through why they are here.

Reply:
Easy enough to hack these out of our live installation and see if we find those labels valuable on not